deployments/arbitrum-goerli/usdc/deploy.ts

```diff
@@ -0,0 +1,69 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet } from '../../../src/deploy';
+
+const HOUR = 60 * 60;
+const DAY = 24 * HOUR;
+
+const GOERLI_TIMELOCK = '0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const trace = deploymentManager.tracer()
+  const ethers = deploymentManager.hre.ethers;
+
+  // pull in existing assets
+  const USDC = await deploymentManager.existing('USDC', '0x72A9c57cD5E2Ff20450e409cF6A542f1E6c710fc', 'arbitrum-goerli');
+  const WETH = await deploymentManager.existing('WETH', '0xe39ab88f8a4777030a534146a9ca3b52bd5d43a3', 'arbitrum-goerli');
```

Comment:
Note: Need to double-check these addresses and deploy an ARB token.

Reply:
I updated these assets in the testnet proposal branch:

https://github.com/compound-finance/comet/pull/740/files#diff-c125ea239f1fd50b6bc7552e42d76f932288f7d50e6124a9f6892028a93c60b2R14

I'm going to clean up the proposal branches and then I'll bring those changes here.